Portfolio.html

```diff
@@ -58,7 +58,225 @@ <h1 class="intro_2">
             <li><a href="#"><img src="github.jpg" alt="github_icon"/></a></li>
             <li><a href="#"><img src="hand_icon.jpg" alt="hand"/></a></li>    
         </ul>
-      </section>           
+      </section> 
+      
+      <section class="works">
+        <article class="card card_1">
+          <div class="img_4"><img class="img" src="Snapshoot_Portfolio.png" alt="snap"/></div>
+
+          <div class="heading"><h1>Tonic</h1></div>
+
+          <div class="card_details">
+            <div class="list">
+            <p>CANOPY</p>
+            <ul>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+            </div>
+
+            <div class="card_text">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            </div>
+
+            <div class="tech">
+            <ul>
+              <li><a href="#">html</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">javascript</a></li>
+            </ul>
+            </div>
+
+            <div class="button">
+              <button>See Project</button>
```

Comment:
The color used when a button is hovered is incorrect.

- [ ] Please fix the hover color here and all other buttons 🙏 
 
Reference | 
------------ | 
![image](https://user-images.githubusercontent.com/67957533/129461722-2a37c5d2-15bc-4e41-bc34-e46352b9877e.png) |

_If you're having trouble getting the color, you have to option to sample colors from webpages using [ColorZilla](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp)_